app/build.gradle

```
@@ -143,14 +142,8 @@ publishing {
             }
         }
 
-        mavenFatJar(MavenPublication) {
-            artifact fatJar
-            artifactId = "corda-cli"
-            version = project.version
-        }
-
-        mavenAppJar(MavenPublication) {
-            artifact jar
+        mavenCordaCLi(MavenPublication) {
+            artifact cordaCLi
             artifactId = "app"
```

Should we be using the default submodule here as the artifact ID 'app' or should it be kept as 'corda-cli'? 
```suggestion
            artifactId = "corda-cli"
```